ports/espressif/boards/m5stack_cardputer/mpconfigboard.mk

```diff
@@ -0,0 +1,13 @@
+USB_VID = 0x303A
+USB_PID = 0x811A
+USB_PRODUCT = "M5STACK CoreS3 - CircuitPython"
```

Comment:
```suggestion
USB_PRODUCT = "M5STACK StampS3 - CircuitPython"
```

Reply:
For the same reason explained below I suggest to use a unique name for Cardputer

`
USB_PRODUCT = "M5STACK Cardputer - CircuitPython"
`

Reply:
I have taken these suggestions, thanks!